### DISCUSSION TITLE
學期總結與抒發

### DISCUSSION THREAD

**Discussion by thz0000** [1101-GS4719-C/Python-Community-GS4719-C#1849] [General]
我的期末考結束了，剩下兩三個報告，上來關心大家的狀況跟抒發一下自己的情緒。

說實話，自己的二上過得不算好，有點忙但又好像是瞎忙居多，讀書的時間也排得不夠多，所以這學期的成績不怎麼好看。

也或許是科目變深了，需要花更多時間和心力去準備，無法如同一年級般考前幾天再認真啃書。

雖然理智知道是自己沒認真讀書才會這樣，但還是有點難過，算是2023的一點小遺憾吧。期許自己之後能多用功，別到最後才自怨自艾，就像今年的自己XD

不知道大家這學期過得好嗎？有沒有發生好玩的事？我的生活有點無聊QQ需要一些其他人的故事哈哈哈

**Comment by Terryyyyyyyyyyyy** [comment]
我的上學期生活也過得很爛，不僅沒選到課，就算有選到，也面臨被當的邊緣，希望我下學期能進步一點，宣導一些輕鬆又分數高的課，也希望能努力一點

**Comment by FakeWood** [comment]
這學期接了系上的一些活動比較忙，感覺書都是在段考前幾天才開始讀的，不過好在這學期的課比較涼，還算可以應付的過去，下學期開始要認真讀書了，不然熬夜實在還是對身體很不好，一起加油!

**Comment by RavenJiaSin** [comment]
學期一開始都會想:這學期要好好念書，成績要進步。然後通常這個想法到學期中就會隨這各種活動而逐漸煙消雲散了XD。但說實在，這學期感覺時間真的好快，可能是我真的參加很多活動。體感2個月的感覺

**Comment by TobywithDino** [comment]
這學期為了兩個段考竟然通宵了兩次，好在結果都感覺不錯，沒有虧待了身體。 但希望下學期可以認真向上好好上課
不要再翹課不聽老師上課了，但相對的，也希望教授的教法可以讓我再更容易理解一點XD

**Comment by chuanmin0317** [comment]
這學期的個科目的段考都不是在同一個禮拜考，所以感覺每次考試周都變成是考試月了然後剛考完沒多久，下一個考試月又來了，感覺這學期都在考試也沒有多餘的時間提升自己，希望下學期能夠安排好自己的時間提升自己的能力

**Comment by lyykey** [comment]
目前正值學期初，我還在當寒假過年過，每天睡到11.起床，梳洗換個衣服就出門吃午餐，接著在悠哉的看今天有沒有跟人有牌局約，沒有的話就會打開電腦，要馬追劇、不然就打遊戲，一天就是這麼的充實，讓我現在大三也不用擔心大學生活沒有快樂到，充實自己，永不放棄，大家加油。

**Comment by benbebobo149** [comment]
我是一個很喜歡挑戰自己的人，這個學期我選了41學分，看起來非常多的，但沒錯，他真的非常多。一開始可能會覺得不太適應，會覺得很累，但習慣之後其實跟平常一樣。希望你到期末的時候，還可以看到我來這裡發言，如果沒有可能要麻煩你來關心我一下。最後，我要感謝同學的分享，讓我有機會了解到這樣有趣的挑戰。

**Comment by YANRONG0123** [comment]
前一個學期我選了25學分，外加一門體育課，雖然相較於其他人，可能相對少了一點，但這讓我生活過得相當充實，學習如何精確的掌握時間，挑戰自己的睡眠極限，以及學習如何團隊合作發揮最高效益。

**Comment by sunset513** [comment]
這個學期我選了24學分，加上體育課，雖然相較於41學分的同學，可能忘是望其項背。但我覺得這是我評估我的實力以及課外活動後，做出的正確判斷。我覺得不能只有課業上的學習，我的生活需要加上各式競賽、工讀、社團、體育、藝文活動，課外學習等等等，這樣的綜合讓我的大學生活豐富又精采。最後感謝版主的分享，給了我一個回顧自己的契機。
